src/bioregistry/data/bioregistry.json

```diff
@@ -53836,6 +53843,96 @@
     "pattern": "^\\d{8}$",
     "uri_format": "http://purl.obolibrary.org/obo/HTN_$1"
   },
+  "hubmap.collection": {
+    "contact": {
+      "email": "katy@iu.edu",
+      "github": "https://github.com/cns-iu/katy",
+      "name": "Katy Borner",
+      "orcid": "0000-0002-3321-6137"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "The collections section of HuBMAP groups related datasets, samples, and metadata into organized categories. Each collection has a unique identifier that links it to its constituent datasets.",
+    "example": "3ae4ddfc175d768af5526a010bfe95aa",
+    "github_request_issue": 1548,
+    "homepage": "https://portal.hubmapconsortium.org/collections",
+    "name": "Human BioMolecular Atlas Program",
+    "pattern": "^[a-f0-9]{32}$",
+    "publications": [
+      {
+        "doi": "10.1038/s41592-024-02563-5",
+        "pmc": "PMC11978508",
+        "pubmed": "40082611",
+        "title": "Human BioMolecular Atlas Program (HuBMAP): 3D Human Reference Atlas construction and usage",
+        "year": 2025
+      }
+    ],
+    "uri_format": "https://portal.hubmapconsortium.org/browse/collection/$1"
+  },
+  "hubmap.donor": {
+    "contact": {
+      "email": "katy@iu.edu",
+      "github": "https://github.com/cns-iu/katy",
+      "name": "Katy Borner",
+      "orcid": "0000-0002-3321-6137"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "The samples subsection of HuBMAP provides metadata about individual tissue samples collected from donors, and are the downstream experimental datasets for scRNA sequencing. Each sample has a unique identifier that links it to donor information, anatomical source, and related datasets.",
```

Comment:
This seems like the wrong description here. This should be about `hubmap.donor`.